lib/runtime-core/src/backing.rs

```diff
@@ -102,6 +103,9 @@ impl LocalBacking {
         memories.into_boxed_map()
     }
 
+    /// Initialize each local memory.
```

Comment:
> Initializes each

Does `local` mean not internal or something else? Could we clarify what local is local to?